libbpf-tools/Makefile

```diff
@@ -59,7 +60,7 @@ $(OUTPUT) $(OUTPUT)/libbpf:
 
 $(APPS): %: $(OUTPUT)/%.o $(LIBBPF_OBJ) $(COMMON_OBJ) | $(OUTPUT)
 	$(call msg,BINARY,$@)
-	$(Q)$(CC) $(CFLAGS) $^ -lelf -lz -o $@
+	$(Q)$(CC) $(CFLAGS) $^ -latomic -lelf -lz -o $@
```

Comment:
why is this needed? those built-ins shouldn't require extra libraries

Reply:
I found when calling   generic version of an atomic operation, such as `__atomic_store` without `_n`, we need to link `libatomic`.